
order: [joint1, joint2, joint3, joint4, joint5, joint6, joint_gripper_left]
sleep: [0, 1.4, -1.47, 0, -0.4851, 0]
singles: []

joint1:
  ID: 1
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 0
  Velocity_Limit: 4096
  Min_Position_Limit: -1145
  Max_Position_Limit: 1145
  Secondary_ID: 255

joint2:
  ID: 2
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 0
  Velocity_Limit: 4096
  Min_Position_Limit: -899
  Max_Position_Limit: 802
  Secondary_ID: 255

joint3:
  ID: 3
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 1
  Velocity_Limit: 4096
  Min_Position_Limit: -847
  Max_Position_Limit: 1031
  Secondary_ID: 255

joint4:
  ID: 4
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 1
  Velocity_Limit: 4096
  Min_Position_Limit: -1661
  Max_Position_Limit: 1661
  Secondary_ID: 255

joint5:
  ID: 5
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 0
  Velocity_Limit: 4096
  Min_Position_Limit: -1031
  Max_Position_Limit: 916
  Secondary_ID: 255

joint6:
  ID: 6
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 0
  Velocity_Limit: 4096
  Min_Position_Limit: -1776
  Max_Position_Limit: 1776
  Secondary_ID: 255

joint_gripper_left:
  ID: 7
  Baud_Rate: 1000000
  Return_Delay_Time: 0
  Drive_Mode: 0
  Velocity_Limit: 4096
  Min_Position_Limit: 588
  Max_Position_Limit: 0
  Secondary_ID: 255



